modules/sso/jetpack-sso-login.js

```diff
@@ -0,0 +1,25 @@
+jQuery( document ).ready( function( $ ) {
+	var ssoWrap = $( '.jetpack-sso-wrap' ),
+		$rememberme = $( '#rememberme' ),
```

Comment:
Maybe change the variable to rememberMe to be more consistent with the rest of the variables.